src/app/routes/testInclude/getInitialData/index.js

```diff
@@ -0,0 +1,75 @@
+import pipe from 'ramda/src/pipe';
+import path from 'ramda/src/path';
+import fetchPageData from '../../utils/fetchPageData';
+import {
+  augmentWithTimestamp,
+  addIdsToBlocks,
+  applyBlockPositioning,
+} from '../../utils/sharedDataTransformers';
+import parseInternalLinks from '../../cpsAsset/getInitialData/convertToOptimoBlocks/blocks/internalLinks';
+import addHeadlineBlock from '../../cpsAsset/getInitialData/addHeadlineBlock';
+import timestampToMilliseconds from '../../cpsAsset/getInitialData/timestampToMilliseconds';
+import addSummaryBlock from '../../cpsAsset/getInitialData/addSummaryBlock';
+import cpsOnlyOnwardJourneys from '../../cpsAsset/getInitialData/cpsOnlyOnwardJourneys';
+import addBylineBlock from '../../cpsAsset/getInitialData/addBylineBlock';
+import addAnalyticsCounterName from '../../cpsAsset/getInitialData/addAnalyticsCounterName';
+import convertToOptimoBlocks from '../../cpsAsset/getInitialData/convertToOptimoBlocks';
+import processUnavailableMedia from '../../cpsAsset/getInitialData/processUnavailableMedia';
+import { MEDIA_ASSET_PAGE } from '#app/routes/utils/pageTypes';
+
+const formatPageData = pipe(
+  addAnalyticsCounterName,
+  parseInternalLinks,
+  timestampToMilliseconds,
+);
+
+export const only = (pageType, transformer) => (pageData, ...args) => {
+  const isCorrectPageType = path(['metadata', 'type'], pageData) === pageType;
+  return isCorrectPageType ? transformer(pageData, ...args) : pageData;
+};
+
+const processOptimoBlocks = pipe(
+  only(MEDIA_ASSET_PAGE, processUnavailableMedia),
+  addHeadlineBlock,
+  addSummaryBlock,
+  augmentWithTimestamp,
+  addBylineBlock,
+  addIdsToBlocks,
+  applyBlockPositioning,
+  cpsOnlyOnwardJourneys,
+);
```

Comment:
The duplication of the pre-proc logic is a bit fragile, but not sure I'd want to extract it out just to facilitate a test page?